x-pack/legacy/plugins/monitoring/public/services/__tests__/executor_provider.js

```diff
@@ -37,24 +37,6 @@ describe('$executor service', () => {
 
   afterEach(() => executor.destroy());
 
-  it('should register listener for fetch upon start', () => {
```

Comment:
Is there a way to replace these tests with newer tests based on the changes?

Reply:
Like what for example?
How would you implement those?

Reply:
@chrisronline I agree with you. 

My changes removed the `listeners` array exposed by `timefilter` and replaced them with `Observables`. 

These tests are checking that listeners were being added to that array (and not any particular side effects of adding those filters), however, since there were no `listeners` anymore, the new implementation wasn't straight forward and felt excessive. I would appreciate if you could open a task to add them, if you still find them useful.

Any other comments on the implementation or we're LGTM? ;-)